src/Templates/AdminPanel/Bit.AdminPanel/Web/App.razor

```diff
@@ -5,9 +5,9 @@
                 AdditionalAssemblies="@lazyLoadedAssemblies"
                 OnNavigateAsync="@OnNavigateAsync">
             <Found Context="routeData">
-                <AuthorizeRouteView RouteData="@routeData">
+                <AuthorizeRouteView Context="authState" RouteData="@routeData">
```

Comment:
please use the AuthenticationState cascading parameter instead of this parameter passing.